src/build/buildWeb.mjs

```diff
@@ -103,16 +113,20 @@ async function copy(copyMap) {
 }
 
 function execPromise(cmd) {
-  return new Promise((r) =>
-    exec(cmd, (_, stdout) => {
-      r(stdout);
+  return new Promise((res, rej) =>
+    exec(cmd, (err, stdout) => {
+      if (err) {
+        console.log(stdout);
+        rej(err);
```

Comment:
Oh you're right - a control flow gotcha! Thanks. While [not strictly needed](https://stackoverflow.com/questions/32536049/do-i-need-to-return-after-early-resolve-reject) here because a rejected promise can't change back to resolved - so the resolve below is a no-op - it is good practice to return, to match the intended control flow.

Reply:
Addressed